tools/yaml-templates/build-test-publish.yml

```diff
@@ -29,8 +29,8 @@ steps:
 
   # Temporarily disable this step until after v2.0.0 is released to avoid having dev releases with inaccurate
   # 2.0.0-beta.x naming
-  # - script: 'node prepNextDevRelease.js'
-  #   displayName: 'node prepNextDevRelease.js'
+  # - script: 'node prepBetaRelease.js'
```

Comment:
I've been running the script and I think there is a problem. The `yarn beachball bump` command needs to be executed from the monorepo root. But the script is executed from the `packages/teams-js` directory, and when run from that directory, it seems like `yarn beachball bump` fails. 

Have you been able to run this script locally successfully?

Reply:
I think I found a way around this (https://github.com/OfficeDev/microsoft-teams-library-js/pull/1193/files#r886228992), but it still doesn't seem to be working properly for me locally.

Reply:
I have been able to run this script locally, but I will look into this more